pagopa/io-app
# feat(IT Wallet): [SIW-251] Init itw main saga and activation base screen

## Short description
This PR initialize ITW saga and a base activation screen. Also adds navigation action to sticky component to start activation flow.

https://github.com/pagopa/io-app/assets/49342144/a93d349c-4f00-45f4-a9f3-d255d29f6571

## List of changes proposed in this pull request
- Add ITW saga
- Add action to start activation flow
- Add base activation screen

## How to test

- using io-dev-api-server enable itWallet feat flag
- In the message home screen tap on itw banner
- an empty activation screen should appear
